test/integration/content_store_organisation_test.rb

```diff
@@ -161,6 +151,6 @@ class OrganisationTest < ActionDispatch::IntegrationTest
     }
     content_store_has_item('/government/organisations/prime-ministers-office-10-downing-street', content_item_with_empty_featured_policies)
     visit "/government/organisations/prime-ministers-office-10-downing-street"
-    assert page.has_css?(".organisation-policies")
+    #assert page.has_css?(".organisation-policies")
```

Comment:
I would probably comment out the whole test so it's more obvious that we're not running this. Or marking it as to skip so we don't run it at all and it flags up as 'pending' in the test run? 

May also be worth noting this in https://trello.com/c/Qw7Y6CKE/150-add-our-policies-list-to-what-we-do-section so the person who works on that card definitely knows this test is available to them.

Reply:
I'll take it out completely - it's something we should cover in future work.